ramls/item.json

```diff
@@ -195,8 +195,9 @@
           "type": "string"
         },
         "date": {
-          "description": "Date of a status",
-          "type": "string"
+          "description": "Date and time when the status was last changed",
+          "type": "string",
+          "format": "date-time"
```

Comment:
This could be considered a breaking compatible change (though, opinions differ as to whether that is practically the case).

@skoczko should this be a regarded as a breaking compatibility change?